### PR TITLE
Adding jekyll pages to generate Dynflow project page / documentation

### PR DESCRIPTION
**Pull request by pitr-ch**
usage:
* `cd pages`
* `bundle install`
* `jekyll serve`
* open `localhost:4000`

### CODE REVIEW

File: pages/_config.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+name: Dynflow
+url: http://blog.pitr.ch
```
Comment (iNecas) [inline pages/_config.yml]: github page url would be better :)

File: pages/_config.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+plugins:      ./plugins
+include:
+- .nojekyll
+
+tag_page_layout: tag_page
+tag_page_dir: tag
+
+permalink: /blog/:year/:month/:day/:title/
+
+disqus_short_name: pitr-ch
```
Comment (iNecas) [inline pages/_config.yml]: do you want to use your accounts here?

Comment (pitr-ch) [inline pages/_config.yml]: we need to fix these things before publishing, for now i left it there

File: pages/.ruby-version [hunk 3]
```diff
@@ -0,0 +1 @@
+2.2.0
```
Comment (iNecas) [inline pages/.ruby-version]: Shouldn't this file got to .gitignore?

File: pages/Gemfile [hunk 4]
```diff
@@ -0,0 +1,12 @@
+source 'https://rubygems.org'
+
+gem 'rake'
+gem 'jekyll'
+gem 'kramdown'
+gem 'pry'
+gem 'ruby-nuggets'
+
+gem 'debase'
```
Comment (iNecas) [inline pages/Gemfile]: We probably don't need all this deps

**Comment by iNecas** [comment]
The font for code samples looks strange on my laptop: increasing the font size to 14px helped.

Overall impress is really great! Looking forward to have finally proper project pages! TY

**Comment by pitr-ch** [comment]
replaced by #138

**Pull request closed without merging**